internal/cmd/billing/command_cost_list.go

```diff
@@ -0,0 +1,110 @@
+package billing
+
+import (
+	"fmt"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+type costOut struct {
+	StartDate           string `human:"Start Date" serialized:"start_date"`
+	EndDate             string `human:"End Date" serialized:"end_date"`
+	Granularity         string `human:"Granularity" serialized:"granularity"`
+	LineType            string `human:"Line Type" serialized:"line_type"`
+	Product             string `human:"Product" serialized:"product"`
+	ResourceId          string `human:"Resource ID" serialized:"resource_id"`
+	ResourceDisplayName string `human:"Resource Display Name" serialized:"resource_display_name"`
+	EnvironmentId       string `human:"Environment ID" serialized:"environment_id"`
+	NetworkAccessType   string `human:"Network Access Type" serialized:"network_access_type"`
+	Price               string `human:"Price" serialized:"price"`
+	OriginalAmount      string `human:"Original Amount" serialized:"original_amount"`
+	DiscountAmount      string `human:"Discount Amount" serialized:"discount_amount"`
+	Amount              string `human:"Amount" serialized:"amount"`
```

Comment:
i assume none of these need omit-empty?

Reply:
(and they should be `float64`)

Reply:
@brianstrauch  would output support `*float64`? That way we could differentiate between 0 value and empty field.

Reply:
@brianstrauch i noticed this issue, it prints out the pointer value instead of the actual value:
![image](https://github.com/confluentinc/cli/assets/13591735/10ed597f-e584-4785-a13f-cf5f3f869d4c)

Reply:
I dont think the `omitempty` works either for floats, as i still see 0 in the test.